app/filters/image_filter.rb

```diff
@@ -1,13 +1,13 @@
 # encoding: utf-8
 
 class ImageFilter < Filter
-
   def process(post)
     parser = Nokogiri::HTML::DocumentFragment.parse(post)
-    parser.css('img').each do |element|
-      if element.attributes && !element.attributes['src'].blank?
-        url = element.attributes['src']
-        if element.attributes['width'].blank? || element.attributes['height'].blank?
+    parser.css("img").each do |element|
+      if element.attributes && !element.attributes["src"].blank?
+        url = element.attributes["src"]
+        if element.attributes["width"].blank? ||
+           element.attributes["height"].blank?
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.